### PR TITLE
docs: adding show-and-tell template to GH discussions

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
## Description

adding show-and-tell template to GH discussions

Note: I am not sure if this is working as expected

### CODE REVIEW

File: .github/DISCUSSION_TEMPLATE/show-and-tell.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+title: "[Show and tell] "
```
Comment (itaysk) [inline .github/DISCUSSION_TEMPLATE/show-and-tell.yml:1]: This means the title of the newly created discussion has this prefix?

[the remark this answers is not in the document]
Comment (itaysk) [reply .github/DISCUSSION_TEMPLATE/show-and-tell.yml:1]: (see my other comment about company name for why this suggestion)

**Review by itaysk — CHANGES_REQUESTED** [review]

File: .github/DISCUSSION_TEMPLATE/show-and-tell.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+labels: ["Share with us your Trivy use cases"]
+body:
+  - type: textarea
+    id: company
```
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply .github/DISCUSSION_TEMPLATE/show-and-tell.yml:5]: People will not know that they should put as title the company -- so I would rather ask explicitly. 
In the discussion YAML itself on the first line, the title has to be "Show and tell" so the form shows up when people select show and tell

File: .github/DISCUSSION_TEMPLATE/show-and-tell.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply .github/DISCUSSION_TEMPLATE/show-and-tell.yml:1]: In the GH documentation on creating a discussion form they explicitly said that the title of the YAML for the form has to match the selection name in the GH discussions of your project -- I think this does not mean that people cannot set a title for their discussion (?)

File: .github/DISCUSSION_TEMPLATE/show-and-tell.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      label: Please select all the scan targets that you are using
+      options:
+        - label: Container Images
+        - label: Filesystem
+        - label: Git Repository
+        - label: Virtual Machine Images
+        - label: Kubernetes
+        - label: AWS
+    validations:
+      required: true
```
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply .github/DISCUSSION_TEMPLATE/show-and-tell.yml:44]: hmm you have a point

File: .github/DISCUSSION_TEMPLATE/show-and-tell.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (itaysk) [reply .github/DISCUSSION_TEMPLATE/show-and-tell.yml:1]: I think you might have confused name of file with name of form?

> You can create custom discussion category forms by adding a YAML form definition file ... The name must correspond with the slug for one of your discussion categories. For example, the template for the "Announcements" category should be .github/DISCUSSION_TEMPLATE/announcements.yml.

File: .github/DISCUSSION_TEMPLATE/show-and-tell.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  - type: textarea
+    id: more
+    attributes:
+      label: description
+      description: "Please tell us more about your use case of Trivy -- anything that you would like to share 🎉"
+      value: |
+        ...
+      render: bash
```
[the remark this answers is not in the document]
Comment (AnaisUrlichs) [reply .github/DISCUSSION_TEMPLATE/show-and-tell.yml:54]: Would you like the type to be markdown?

[the remark this answers is not in the document]
Comment (itaysk) [reply .github/DISCUSSION_TEMPLATE/show-and-tell.yml:44]: That's also not desired (will render as markdown codeblock). Just remove it

**Review by itaysk — APPROVED** [review]

**Comment by itaysk** [comment]
@knqyf263 you're required as code owner

**Comment by knqyf263** [comment]
Did we test this template?

**Comment by AnaisUrlichs** [comment]
@knqyf263 it does not really fall under this any of those types tho :D will call it "docs"

**Comment by knqyf263** [comment]
@itaysk @AnaisUrlichs Did either of you two confirm the template actually work in fork or another repository?

**Comment by AnaisUrlichs** [comment]
@knqyf263 will do

**Comment by itaysk** [comment]
@AnaisUrlichs can you also please also add an optional field to provide a link to logo? maybe in the future we can create a page like this in the Trivy website: https://fluxcd.io/adopters/

**Comment by AnaisUrlichs** [comment]
@knqyf263 @itaysk I made some changes and checked that the template works on my branch -- please let me know if you would like anything else to be changed
Screenshot 2023-01-22 at 14 34 39

**Review by knqyf263 — APPROVED** [review]
Looks great to me!

**Pull request merged**